### DISCUSSION TITLE
This is a chat model and not supported API response

### DISCUSSION THREAD

**Discussion by ghost** [waylaidwanderer/node-chatgpt-api#159] [Help] [answered]
Hi

I am using chatgpt-cli.
When I use the model 'text-davinci-003; in settings.js - everything works perfectly. 

When I use the model 'gpt-3.5-turbo'
I get the following error :
"This is a chat model and not supported in the v1/completions endpoint. Did you mean to use v1/chat/completions?"

Main Branch Code - Sunday 5th March.

Does it mean my account doesn't have access to the new "Chat API"? note - I can use the "Chat API" in the official playground. I can't see any other syntax or other error -if so 'text-davinci-003' won't work.

Any ideas?

**Comment by waylaidwanderer** [comment, marked as answer]
I'm guessing your local repo isn't up to date because that's impossible if you're setting `modelOptions.model` to `gpt-3.5-turbo` - my code explicitly checks for that.

```
# Make sure you are on the main branch
git checkout main

# Fetch the latest changes from the remote repository
git fetch origin main

# Merge the changes from the remote main branch into your local main branch
git merge origin/main
```

**Reply by ghost** [reply]
I am happy to send you debug = true if you like?

**Reply by ghost** [reply]
Here is the debug screenshot.. 

![Screenshot 2023-03-05 at 7 58 56 pm](https://user-images.githubusercontent.com/91181815/222951324-85e00720-d30b-40f1-a17a-645d5a0a6789.png)

**Reply by waylaidwanderer** [reply]
@99yarns thanks. To confirm you're on the latest commit, could you paste the `git rev-parse HEAD` output? It should be `011b0c4a54d5931d70895c037d829b32dca5f1ca` if you are.

**Reply by waylaidwanderer** [reply]
Wait.

You said:
>Main Branch Code - Sunday 5th March.

I took this to mean that you cloned my repo (`git clone <URL>`) and are running it directly.

But your debug log says `file:///usr/local/lib/node_modules/@waylaidwanderer/chatgpt-api/src/ChatGPTClient.js:93:41` in the stack trace. This suggests that you installed my module globally using `npm i -g @waylaidwanderer/chatgpt-api`. **You need to update the module to the latest version.**

**Reply by ghost** [reply]
THANK YOU!!!! :)

I had an older version of chatgpt-api node module in my /usr/local/lib. 

All is working :)
